packages/experimental/RadioGroup/src/RadioGroup/useRadioGroup.ts

```diff
@@ -20,13 +20,15 @@ export const useRadioGroup = (props: RadioGroupProps): RadioGroupInfo => {
 
   const state: RadioGroupState = {
     value: data.selectedKey,
+    disabled: disabled || false,
     onChange: data.onKeySelect,
     updateSelectedButtonRef: onSelectButtonRef,
   };
 
   return {
     props: {
       ...props,
+      disabled,
       accessible: true,
       accessibilityRole: 'radiogroup',
       accessibilityLabel: accessibilityLabel ?? label,
```

Comment:
I believe that we need to set accessibilityState as disabled for property accessibility support. Same with individual radios

Reply:
Added!